### DISCUSSION TITLE
Webp Cropped quality change?

### DISCUSSION THREAD

**Discussion by wes-kay** [ImageMagick/ImageMagick#4249] [Help]
I'm resizing a webp image `magick convert img.webp -crop x +repage output.webp` but you can see in the image I'm getting some issues in the quality.
![Screenshot_86](https://user-images.githubusercontent.com/40011470/133918252-979d4c5f-8af5-4d37-9ff1-cc436709592e.png)
.

**Comment by snibgo** [comment]
What version of IM, on what platform (bash, Windows, etc)? Is webp an internal or external delegate? What is the full text output from "magick -version"?

You say you are resizing an image, but your command has no "-resize".

I don't use webp much, but it seems to be 8 bits/channel/pixel, and by default it is lossy. "-quality 100" seems to make it lossless, for me (but still only 8 bit).

**Reply by wes-kay** [reply]
Sorry that's my mistake, it was a late night, it was in fact a crop I'm doing. 

```
Version: ImageMagick 7.0.11-6 Q16 x64 2021-04-03 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Visual C++: 192829912
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raw rsvg tiff webp xml zip zlib
```

**Comment by snibgo** [comment]
For you, webp is a buit-in (ie internal) delegate. Same here.

Does "-quality 100" solve your problem? If not, please post your webp input and output (perhaps in a zip file that you upload to somewhere and paste the link here), and show the IM command you used.
